### PR TITLE
[FR] Update Twitter

### PR DESCRIPTION
**Pull request by cl8n**
#2543

### CODE REVIEW

File: wiki/Twitter/fr.md [hunk 1]
```diff
@@ -1,16 +1,13 @@
----
-outdated: true
----
-
 # Twitter
 
 ## Nous tweetons aussi!
 
-osu! a plusieurs comptes Twitter, chacun sert à quelque chose de différent. La plupart ne tweetent qu'un peu mais vous gardent au courant de ce qui se passe dans l'écosystème d'osu!
+osu! a plusieurs comptes Twitter, chacun servant des objectifs spécifiques. La plupart sont très peu fréquentés mais vous aideront à vous tenir informés en dehors de l'osu! écosystème.
```
Comment (Pachiru) [inline wiki/Twitter/fr.md]:
```suggestion
osu! possède plusieurs comptes Twitter, chacun ayant des objectifs spécifiques. La plupart sont très peu fréquentés mais vous aideront à vous maintenir informés en dehors de l'écosystème d'osu!.
```

File: wiki/Twitter/fr.md [hunk 2]
```diff
@@ -1,16 +1,13 @@
...
-| ![Dean Herbert avatar](img/ppy.jpg) | [@ppy](https://twitter.com/ppy) | Pas strictement lié à osu!, mais je le laisse ici pour ceux qui sont intéressés. Ma vie c'est osu!, après tout. |
-| ![BanchoBoat avatar](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat) | Un élan comique quand les choses se passent mal. |
+| ![osu! avatar](img/osugame.jpg) | [@osugame](https://twitter.com/osugame) | La source officielle de nouvelles et d'annonces. Contenu de haute qualité à faible trafic. |
+| ![osu!status avatar](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus) | Dédié à fournir des notifications à faible trafic des problèmes de site et de bancho. |
+| ![Dean Herbert avatar](img/ppy.jpg) | [@ppy](https://twitter.com/ppy) | Pas strictement osu! liés, mais nous allons laisser cela ici pour ceux qui sont intéressés. La vie de ppy est surtout osu!, après tout. |
+| ![osu! support avatar](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport) | Vous aider avec les problèmes de compte/communauté. |
+| ![BanchoBoat avatar](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat) | Comédie soulagement quand les choses tournent mal. |
```
Comment (Pachiru) [inline wiki/Twitter/fr.md]:
```suggestion
| ![BanchoBoat avatar](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat) | Un élan comique quand les choses se passent mal. |
```

File: wiki/Twitter/fr.md [hunk 3]
```diff
@@ -1,16 +1,13 @@
...
+osu! a plusieurs comptes Twitter, chacun servant des objectifs spécifiques. La plupart sont très peu fréquentés mais vous aideront à vous tenir informés en dehors de l'osu! écosystème.
 
 | Avatar | Compte | Description |
 | :-: | :-: | --- |
-| ![osu! avatar](img/osugame.jpg) | [@osugame](https://twitter.com/osugame) | Un flux des actualités et des notifications des [beatmaps récemment rankées](https://osu.ppy.sh/p/beatmaplist). |
-| ![osu!status avatar](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus) | Dédié à fournir de rares notifications sur les problèmes du site et de bancho. |
-| ![Dean Herbert avatar](img/ppy.jpg) | [@ppy](https://twitter.com/ppy) | Pas strictement lié à osu!, mais je le laisse ici pour ceux qui sont intéressés. Ma vie c'est osu!, après tout. |
-| ![BanchoBoat avatar](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat) | Un élan comique quand les choses se passent mal. |
+| ![osu! avatar](img/osugame.jpg) | [@osugame](https://twitter.com/osugame) | La source officielle de nouvelles et d'annonces. Contenu de haute qualité à faible trafic. |
```
Comment (Pachiru) [inline wiki/Twitter/fr.md]:
```suggestion
| ![osu! avatar](img/osugame.jpg) | [@osugame](https://twitter.com/osugame) | Un flux des actualités et des notifications des [beatmaps récemment rankées](https://osu.ppy.sh/p/beatmaplist) |
```

File: wiki/Twitter/fr.md [hunk 4]
```diff
@@ -1,16 +1,13 @@
...
+| ![osu!status avatar](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus) | Dédié à fournir des notifications à faible trafic des problèmes de site et de bancho. |
```
Comment (Pachiru) [inline wiki/Twitter/fr.md]:
```suggestion
| ![osu!status avatar](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus) | Dédié à fournir de rares notifications sur les problèmes du site et de bancho. |
```

File: wiki/Twitter/fr.md [hunk 5]
```diff
@@ -1,16 +1,13 @@
...
+| ![Dean Herbert avatar](img/ppy.jpg) | [@ppy](https://twitter.com/ppy) | Pas strictement osu! liés, mais nous allons laisser cela ici pour ceux qui sont intéressés. La vie de ppy est surtout osu!, après tout. |
```
Comment (Pachiru) [inline wiki/Twitter/fr.md]:
```suggestion
| ![Dean Herbert avatar](img/ppy.jpg) | [@ppy](https://twitter.com/ppy) | Pas strictement lié à osu!, mais je le laisse ici pour ceux qui sont intéressés. Ma vie c'est osu!, après tout. |
```

File: wiki/Twitter/fr.md (hunk 3 above)
Comment (cl8n) [reply wiki/Twitter/fr.md]: if you want to change the content/meaning you should either change EN first or open an issue

**Pull request merged**